tests/OSPSuite.Core.Tests/Extensions/ModuleNodeExtensionsTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using NUnit.Framework;
+using OSPSuite.BDDHelper;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+
+namespace OSPSuite.Core.Extensions
+{
+   public abstract class concern_for_ModuleExtensions : StaticContextSpecification
+   {
+      protected Module _module;
+      protected MoleculeBuildingBlock _moleculeBuildingBlock;
+      protected PassiveTransportBuildingBlock _passiveTransportBuildingBlock;
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         _module = new Module();
+         _moleculeBuildingBlock = new MoleculeBuildingBlock();
+         _passiveTransportBuildingBlock = new PassiveTransportBuildingBlock();
+      }
+   }
+
+   [TestFixture]
+   public class ModuleNodeExtensionsTests : concern_for_ModuleExtensions
+   {
+
+      [Observation]
+      public void CanAdd_ShouldReturnTrue_WhenBuildingBlockTypeIsNotInUniqueTypes()
+      {
+         var customBuildingBlock = new CustomBuildingBlock();
+
+         var result = _module.CanAdd(customBuildingBlock);
+
+         Assert.IsTrue(result);
+      }
+
+      [Observation]
+      public void CanAdd_ShouldReturnTrue_WhenNoSameTypeBuildingBlockExists()
+      {
+         var result = _module.CanAdd(_moleculeBuildingBlock);
+
+         Assert.IsTrue(result);
```

Comment:
Could you write this as

`_module.CanAdd(_moleculeBuildingBlock).ShouldBeTrue();`

The OSPSuite.BDDHelper project has a lot of methods and extensions that are set up to make tests easy to read like this.

Follow for the other `Assert` you are using.